Microsoft.Toolkit.Services/Services/Bing/BingService.cs

```diff
@@ -13,6 +14,7 @@ namespace Microsoft.Toolkit.Services.Bing
     /// <summary>
     /// Class for connecting to Bing.
     /// </summary>
+    [Obsolete("The Bing Service will be removed in a future major release. Please use the Cognitive services.")]
```

Comment:
Could you add a link to point to the service?

Reply:
I investigated how to add a link to the Obsolete attribute but I didn't find anything. How could I do it?

Reply:
You could just add it as text. I can create a shortlink if you'd like, what's the link to the service?

Reply:
Sure, the link is: https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Search
Thanks

Reply:
Thanks! I just uploaded it.